File: c#/1038.cs
using System; 
using System.Globalization;

class URI {

    static void Main(string[] args) { 

        string input_iq = Console.ReadLine();
        
        string[] iq = input_iq.Split(' ');
        
        int i = int.Parse(iq[0], CultureInfo.InvariantCulture);
        int q = int.Parse(iq[1], CultureInfo.InvariantCulture);
        
        double T;
        
        if (i == 1) {
            T = 4.00 * q;
        } else if (i == 2) {
            T = 4.50 * q;
        } else if (i == 3) {
            T = 5.00 * q;
        } else if (i == 4) {
            T = 2.00 * q;
        } else {
            T = 1.50 * q;
        }
        
        Console.WriteLine($"Total: R$ {T.ToString("F2", CultureInfo.InvariantCulture)}");

    }

}

File: c++/1038.cpp
#include <stdio.h>
 
int main() {

    int a, b;
    double c;

    scanf("%d", &a);
    scanf("%d", &b);

    if (a == 1) {
        c = b * 4.00;
    } else if (a == 2) {
        c = b * 4.50;
    } else if (a == 3) {
        c = b * 5.00;
    } else if (a == 4) {
        c = b * 2.00;
    } else {
        c = b * 1.50;
    }
    
    printf("Total: R$ %.2lf\n", c);
 
    return 0;
}
